relays/ethereum/Cargo.toml

```diff
@@ -10,11 +10,11 @@ ansi_term = "0.12"
 async-std = "1.9.0"
 async-trait = "0.1.42"
 clap = { version = "2.33.3", features = ["yaml"] }
-codec = { package = "parity-scale-codec", version = "1.3.4" }
+codec = { package = "parity-scale-codec", version = "2.0.0" }
 env_logger = "0.8.2"
-ethabi = "13.0"
-ethabi-contract = "11.0"
-ethabi-derive = "13.0"
+ethabi = { git = "https://github.com/paritytech/ethabi.git", branch = "td-eth-types-11" }
+ethabi-contract = { git = "https://github.com/paritytech/ethabi.git", branch = "td-eth-types-11" }
+ethabi-derive = { git = "https://github.com/paritytech/ethabi.git", branch = "td-eth-types-11" }
```

Comment:
So I guess, this is in order to be compatibel with the codec update, right?

Reply:
Yes, upstream PR is here https://github.com/rust-ethereum/ethabi/pull/204